packages/elements/src/react/sign-in/root.tsx

```diff
@@ -62,22 +63,29 @@ function SignInFlowProvider({ children, exampleMode }: SignInFlowProviderProps)
 }
 
 export type SignInRootProps = SignInFlowProviderProps & {
+  /**
+   * Fallback markup to render while Clerk is loading
+   */
   fallback?: React.ReactNode;
   /**
-   * The base path for your sign-in route. Defaults to `/sign-in`.
-   *
-   * TODO: re-use usePathnameWithoutCatchAll from the next SDK
+   * The base path for your sign-in route.
+   * Will be automatically inferred in Next.js.
```

Comment:
In the future we can adjust this line e.g. with Remix once we support this in other frameworks, too